tests/lib/code-path-analysis/code-path-analyzer.js

```diff
@@ -16,7 +16,7 @@ var assert = require("assert"),
     fs = require("fs"),
     path = require("path"),
     eslint = require("../../../lib/eslint"),
-    EventGeneratorTester = require("../../../lib/testers/event-generator-tester"),
+    EventGeneratorTester = require("../../fixtures/event-generator-tester"),
```

Comment:
K. I will move it back to its original location and annotate the parts that are unused.